help/en/package/jmri/jmrit/operations/Operations.shtml

```diff
@@ -3450,12 +3450,22 @@ Command sequences should also be enclose in "strong" formatting.    jerryg2003 2
       train's route the car will be pulled and eventually dropped off by the train to the car's
       destination and track.
 
-      <p>The quick load service feature allows the "turn" or another train to also pull the car.
-        For the first legs of a train's route carrying the car, the program will create a "clone"
-        car. The clone cars are used to create the train Manifests with a "car" having multiple set
-        outs and pulls. The last leg has the actual car. You need to be careful in the order you
-        reset trains if you allow two or more trains to carry the same car. Always reset the last
-        train built, and work you way backwards reseting in the reverse order you built your trains.</p>
+      <p>
+        The quick load service feature allows the "turn" or another train to also pull the car. For
+        the first legs of a train's route carrying the car, the program will create a "clone" car.
+        The clone cars are used to create the train Manifests with a "car" having multiple set outs
+        and pulls. The last leg has the actual car. If you don't want the train delivering the car
+        to immediate pull the car, set the wait value in the optional <a href="#Schedules">schedule</a>
+        to 1. This will allow the next train built's Manifest to show pulling the car before the car is
+        actually delivered.
+      </p>
+
+      <p>When using this feature, you need to be careful in the order you reset trains if you
+        allow two or more trains to carry the same car. Always reset the last train built, and work
+        your way backwards reseting in the reverse order you built your trains. The other issue is
```

Comment:
I'm not a native English speaker, but I think `reseting` should be `resetting`.

Reply:
Will fix in the next commit

Reply:
Thanks